ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -464,6 +464,13 @@ static Schema resolveArithmeticType(final Schema.Type left,
     return schema;
   }
 
+  static boolean isNumber(final Schema.Type type) {
+    return type == Schema.Type.INT32
+        || type == Schema.Type.INT64
+        || type == Schema.Type.FLOAT64
```

Comment:
Currently we only support the above three numerical types. We can update this when we add the other types.
I agree it would be better to have this in Connect but changing connect would take longer time so I'll keep it here for now.

Reply:
If we don't raise a PR against connect it'll never be in there.  So... do you fancy raising a PR into Connect? :D